crates/ruff_python_parser/tests/snapshots/invalid_syntax@statements__with__unclosed_ambiguous_lpar_eof.py.snap

```diff
@@ -15,7 +15,7 @@ Module(
                     is_async: false,
                     items: [
                         WithItem {
-                            range: 6..6,
+                            range: 5..6,
```

Comment:
It seems that this PR changed whether the with item includes the range of the parentheses. Is this expected?

Reply:
Yes. This is for the following code:
```py
with (
```

Here, the speculative parsing fails and thus it becomes a parenthesized expression.